NIPTool/parse/batch.py

```diff
@@ -25,43 +26,42 @@ def form(val: Optional, function) -> Optional:
 
 
 def validate(key: str, val: Optional) -> Optional:
-    """Formating value according to defined models."""
+    """Formatting value according to defined models."""
 
     if val in exceptions:
-        formated_value = None
+        formatted_value = None
     elif key in ints:
-        formated_value = form(val, int)
+        formatted_value = form(val, int)
     elif key in floats:
-        formated_value = form(val, float)
+        formatted_value = form(val, float)
     elif key in strings:
-        formated_value = form(val, str)
+        formatted_value = form(val, str)
     else:
-        formated_value = None
-    return formated_value
+        formatted_value = None
+    return formatted_value
 
 
 def parse_batch_file(nipt_results_path: str) -> List[dict]:
-    """Parsing file content. Formating values. Ignoring values 
+    """Parsing file content. Formatting values. Ignoring values
     that could not be formatted according to defined models"""
 
     file = Path(nipt_results_path)
 
     if not file.exists():
         raise MissingResultsError("Results file missing.")
 
-    with open(file) as nipt_results_path:
-        df = pd.read_csv(file, na_filter=False)
-        results = df.to_dict(orient="records")
+    df = pd.read_csv(file, na_filter=False)
```

Comment:
Perhaps rename df and pd to something else so it's more clear what they are?

Reply:
👍